README.adoc

```diff
@@ -115,3 +115,31 @@ The places where you must update those sources are then, respectively:
 If you intend on using `hack/operate.sh` it expects you to be in a development environment. Operator installation from this script therefore expects access to the internet. This comes with one extra concern: If `kustomize` isn't in your path, it tries to download it from the internet and save it locally into a `.gitignore`d folder. If you intend on using `hack/operate.sh` to install the operator, you should also bring `kustomize` and place it in the `$PATH` of the user who will be running the script. Additionally, in order to install the operator with `hack/operate.sh` you'll need to make the following change:
 
 . `hack/operate.conf`: IMG should point to the gitea-operator image in your environment
+
+== Adding an additional Certificate Authority trust bundle
+
+The operator can mount an additional trust bundle `ConfigMap` into the Gitea pod. Create your `ConfigMap` and set the `spec.gitea.trustBundleConfigMap` parameter to the name of the `ConfigMap`.
```

Comment:
Should note either here or on the next paragraph that it needs to be in the same workspace as the CR to get picked up like this. It may be unobvious to someone following documentation and creating their configmaps in openshift-config or openshift-ingress-operator for example.

Reply:
Done! :)